### PR TITLE
Submit Form Alignment Fix

### PR DESCRIPTION
**Pull request by nachiketkanore**
Changed the CSS properties for the alignment issue mentioned in #2171

### CODE REVIEW

**Comment by kiritofeng** [comment]
Can you please attach a screenshot of the before/after?

**Comment by nachiketkanore** [comment]
image
image

**Comment by Ninjaclasher** [comment]
What does the submit bar and language select look like?

**Comment by kiritofeng** [comment]
It seems the grey border doesn't cover the corner:
image
Can you see if this can be fixed?

**Comment by nachiketkanore** [comment]
New screenshots
image
zoomed in:
image

File: resources/select2/_multiple.scss [hunk 1]
```diff
@@ -81,6 +81,10 @@
 }
 
 &.select2-container--focus {
+    width: 100%;
```
Comment (Ninjaclasher) [inline resources/select2/_multiple.scss:84]: Let's ... not change values here, as they affect _all_ select2 widgets across the entire site. If they are necessary, add them in `problem.scss` inside i.e. `#problem-submit`.

File: resources/problem.scss [hunk 2]
```diff
@@ -237,13 +237,14 @@ ul.problem-list {
...
         float: right;
+        padding-right: 6px;
+        padding-bottom: 6px;
     }
 
     #submit-wrapper {
-        margin-top: 0.7em;
+        margin-top: .2em;
 
         #editor, #language {
```
Comment (Ninjaclasher) [inline resources/problem.scss:247]: This block is empty, just delete.

File: resources/base.scss [hunk 3]
```diff
@@ -86,7 +86,6 @@ th {
 .form-area {
     display: inline-block;
     background: $color_primary5;
-    padding: 5px 10px 10px;
```
Comment (Ninjaclasher) [inline resources/base.scss:89]: Again, let's not change it globally unless you're sure this won't adversely affect any other `form-area` across the entire site.

**Pull request closed without merging**